.circleci/config.yml

```diff
@@ -355,28 +355,28 @@ workflows:
               # - test_cases
             filters:
               branches:
-                only: /^(master|live)$/
+                only: /^(master|live|GCW-2980-account-details-page-not-visible-ie)$/
         - www-deploy:
             requires:
               - www_build
             filters:
               branches:
-                only: /^(master|live)$/
+                only: /^(master|live|GCW-2980-account-details-page-not-visible-ie)$/
```

Comment:
Make sure you will remove `GCW-2980-account-details-page-not-visible-ie` before merging PR.